tsl/test/expected/transparent_decompression_queries.out

```diff
@@ -0,0 +1,55 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+--TEST github issue 1650 
+CREATE TABLE test_x_hc ( job_run_id INTEGER NOT NULL, mac_id CHAR(16) NOT NULL, rtt INTEGER NOT NULL, ts TIMESTAMP(3) NOT NULL );
+SELECT create_hypertable('test_x_hc', 'ts', chunk_time_interval => interval '1 day', migrate_data => true);
+   create_hypertable    
+------------------------
+ (1,public,test_x_hc,t)
+(1 row)
+
+insert into test_x_hc
```

Comment:
Suggest `INSERT INTO foo VALUES` for consistency.

Reply:
modified tab name and fixed commit msg typo.